### ISSUE TITLE
Critical dependencies:

### ISSUE THREAD

**Issue by mehmobrams** [proj4js/proj4js#215]
WARNING in ./~/proj4leaflet/~/proj4/dist/proj4.js
Critical dependencies:
1:407-414 This seems to be a pre-built javascript file. Though this is possible, it's not recommended. Try to require the original source to get better results.
 @ ./~/proj4leaflet/~/proj4/dist/proj4.js 1:407-414

**Comment by ahocevar** [comment]
Are you using jam? If not, just `require('proj4')`, and the warning will go away.

**Comment by mehmobrams** [comment]
Thanks for your help. I put the require and still have this message.
I am using npm (node).

**Comment by calvinmetcalf** [comment]
no, what bundler are you using, that warning is not somethign generated by node

**Comment by mehmobrams** [comment]
I am using IntelliJ + angular2
Here is my package.json:

{
  "name": "myapp-seed",
  "version": "1.0.0",
  "description": "Polyalert seed based on webpack",
  "scripts": {
    "lint": "tslint \"src/**/*.ts\"",
    "build": "rimraf dist && webpack --config ./config/webpack.prod.js --display-error-details --colors --progress --profile --bail",
    "start": "webpack-dev-server --inline --progress --display-error-details",
    "test": "karma start",
    "pree2e": "webdriver-manager update",
    "e2e": "protractor"
  },
  "private": true,
  "dependencies": {
    "@angular/common": "2.0.2",
    "@angular/compiler": "2.0.2",
    "@angular/core": "2.0.2",
    "@angular/forms": "2.0.2",
    "@angular/http": "2.0.2",
    "@angular/platform-browser": "2.0.2",
    "@angular/platform-browser-dynamic": "2.0.2",
    "@angular/router": "3.0.2",
    "@types/proj4": "^2.3.2",
    "bootstrap": "^3.3.7",
    "core-js": "^2.4.1",
    "leaflet": "^1.0.1",
    "leaflet-draw": "^0.4.1",
    "proj4leaflet": "^0.7.2",
    "rxjs": "5.0.0-beta.12",
    "zone.js": "^0.6.25"
  },
  "devDependencies": {
    "@types/jasmine": "^2.2.30",
    "@types/node": "^6.0.42",
    "@types/core-js": "^0.9.34",
    "@types/leaflet": "^1.0.36",
    "angular-cli": "1.0.0-beta.17",
    "angular2-template-loader": "^0.5.0",
    "awesome-typescript-loader": "^2.2.4",
    "css-loader": "^0.25.0",
    "extract-text-webpack-plugin": "^1.0.1",
    "file-loader": "^0.9.0",
    "html-loader": "^0.4.4",
    "html-webpack-plugin": "^2.15.0",
    "jasmine-core": "^2.4.1",
    "jasmine-spec-reporter": "2.5.0",
    "karma": "^1.3.0",
    "karma-jasmine": "^1.0.2",
    "karma-chrome-launcher": "^2.0.0",
    "karma-phantomjs-launcher": "^1.0.2",
    "karma-sourcemap-loader": "^0.3.7",
    "karma-webpack": "^1.8.0",
    "null-loader": "^0.1.1",
    "phantomjs-prebuilt": "^2.1.7",
    "raw-loader": "^0.5.1",
    "rimraf": "^2.5.4",
    "style-loader": "^0.13.1",
    "tslint": "3.15.1",
    "codelyzer": "~0.0.28",
    "typescript": "^2.0.2",
    "protractor": "^4.0.9",
    "ts-node": "^1.4.3",
    "webpack": "^1.13.2",
    "webpack-dev-server": "^1.16.2",
    "webpack-merge": "^0.14.1"
  }
}

**Comment by calvinmetcalf** [comment]
This is a bug in Webpack where they use the bower.json before the package.json and then complain about bower pointing at a pre built file despite that being what you are supposed to do with bower. 

Tl;Dr you are building your app with Webpack and this is a bug in Webpack.

**Comment by mehmobrams** [comment]
Thanks. Is there any workaround ?

**Comment by calvinmetcalf** [comment]
just ignore the spurious warning?
